community/www/home/index.html

```diff
@@ -10,7 +10,7 @@
 <section id="hero">
   <div class="container">
     <div class="jumbotron">
-      <h1 class="display-4">Guided online courses, with a mentor at your back.</h1>
+      <h1 class="display-4">Guided online courses, with a <br />mentor at your back.</h1>
```

Comment:
Can you achieve this by setting a width to the display instead of <br/>.  This message may change and it would be difficult to figure out where to add the line break.

What happens when you see this on a mobile?